scss/_buttons.scss

```diff
@@ -232,23 +232,28 @@
 
 // Boosted mod: social button
 .btn-social {
-  border-color: currentcolor;
+  --#{$prefix}btn-border-color: currentcolor;
+  --#{$prefix}btn-hover-color: #{$white};
+  --#{$prefix}btn-hover-bg: var(--#{$boosted-prefix}network-color, $black);
+  --#{$prefix}btn-hover-border-color: var(--#{$boosted-prefix}network-color, $black);
+  --#{$prefix}btn-active-color: #{$white};
+  --#{$prefix}btn-active-bg: #{$black};
+  --#{$prefix}btn-active-border-color: #{$black};
+  --#{$prefix}btn-disabled-color: #{$gray-500};
+  --#{$prefix}btn-disabled-bg: transparent;
   @include border-radius(50%, 50%);
   @include button-icon(var(--#{$boosted-prefix}network-logo));
 
-  &:hover,
   &:focus {
-    color: $white;
-    background-color: var(--#{$boosted-prefix}network-color, $black);
-    border-color: var(--#{$boosted-prefix}network-color, $black);
     outline-color: var(--#{$boosted-prefix}network-color, $black);
   }
 
-  &:active,
-  &.active {
-    color: $white;
-    background-color: $black;
-    border-color: $black;
+  &.btn-inverse {
+    --#{$prefix}btn-color: #{$white};
+    --#{$prefix}btn-active-color: #{$black};
+    --#{$prefix}btn-active-bg: #{$white};
```

Comment:
Why not `btn-active-bg-color` instead of `btn-active-bg`?

Reply:
Same as above.